test/tests/provider_function/expected/main.terratag.tf

```diff
@@ -0,0 +1,26 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "5.68.0"
+    }
+  }
+}
+
+resource "aws_ecr_repository" "hashicups" {
+  name = "hashicups"
```

Comment:
Why is it called `hashicups` ? related to the [hashicups provider?](https://registry.terraform.io/providers/hashicorp/hashicups/latest/docs)

Reply:
I just took the example from the offical website...